features/logging/cluster_logging_operator.feature

```diff
@@ -152,3 +152,77 @@ Feature: cluster-logging-operator related test
       | card_name | #{cb.card} |
     Then the step should succeed
     """
+
+  # @author gkarager@redhat.com
+  # @case_id OCP-33868
+  @admin
+  @destructive
+  Scenario: Expose more fluentd knobs to support optimizing fluentd for different environments - Invalid Values
+    Given I register clean-up steps:
+    """
+    Given I delete the clusterlogging instance
+    Then the step should succeed
+    """
+    Given I obtain test data file "logging/clusterlogging/cl_fluentd-buffer_Invalid.yaml"
+    When I run the :create client command with:
+      | f | cl_fluentd-buffer_Invalid.yaml |
+    Then the step should fail
+
+  # @author gkarager@redhat.com
+  # @case_id OCP-33793
+  @admin
+  @destructive
+  Scenario: Expose more fluentd knobs to support optimizing fluentd for different environments
+    Given I obtain test data file "logging/clusterlogging/cl_fluentd-buffer.yaml"
+    And I create clusterlogging instance with:
+      | remove_logging_pods | true                   |
+      | crd_yaml            | cl_fluentd-buffer.yaml |
+    Then the step should succeed
+    Given I wait for the "fluentd" daemon_set to appear up to 300 seconds
+    And <%= daemon_set('fluentd').replica_counters[:desired] %> pods become ready with labels:
+      | logging-infra=fluentd |
```

Comment:
These steps are not needed, because we already check all the EFK pods status in step `I create clusterlogging instance with:` , see https://github.com/openshift/verification-tests/blob/master/features/step_definitions/logging.rb#L273-L274 and https://github.com/openshift/verification-tests/blob/master/features/step_definitions/logging.rb#L291-L302 .